crate/config/src/error/result.rs

```diff
@@ -0,0 +1,45 @@
+use std::fmt::Display;
+
+use super::FindexConfigError;
+
+pub(crate) type FindexConfigResult<R> = Result<R, FindexConfigError>;
+
+#[allow(dead_code)]
+pub(crate) trait ConfigResultHelper<T> {
+    fn context(self, context: &str) -> FindexConfigResult<T>;
+    fn with_context<D, O>(self, op: O) -> FindexConfigResult<T>
+    where
+        D: Display + Send + Sync + 'static,
```

Comment:
Why do you need async properties here? It seems all your functions are sync.

Reply:
right. And deleting the result.rs file since not actually used anywhere...